pkgs/top-level/all-packages.nix

```diff
@@ -9635,6 +9635,8 @@ in
     stdenv = if stdenv.cc.isClang then llvmPackages_6.stdenv else stdenv;
   }) php74 php73 php72;
 
+  php-pearweb-phars = callPackage ../development/interpreters/php/pearweb-phars.nix { };
```

Comment:
This would most probably never be needed outside of the php derivation.

So we can do the `callPackage` in the let block of the `development/interpreters/php/default.nix` file instead. And we already have `callPackage` in there.

Reply:
Fixed